yaml.opam

```diff
@@ -11,7 +11,7 @@ depends: [
   "dune" {build & >="1.3"}
   "ctypes" {>= "0.12.0"}
   "ppx_sexp_conv" {>= "v0.9.0"}
-  "sexplib"
+  "sexplib0" {< "v0.12"}
```

Comment:
I thinkkkkkkk the `< "v0.12"` isnt needed any more due to https://github.com/ocaml/opam-repository/pull/13674 being merged

Reply:
That's great! I'll remove this, hope it works.